pkg/controller/provider/model/vsphere/tree.go

```diff
@@ -45,25 +44,18 @@ func (r InvalidKindError) Error() string {
 type Tree struct {
 	// DB connection.
 	DB libmodel.DB
-	// Tree root.
-	Root Model
-	// Leaf kind.
-	Leaf string
```

Comment:
Now that tree traversal is both down and up, the Root and Leaf only pertain to the `Build()`.